### PR TITLE
FIX: doc-string - langchain.vectorstores.dashvector.DashVector

### PR DESCRIPTION
**Pull request by linzeyang**
- **Description:** There are several mistakes in the sample code in the doc-string of `DashVector` class, and this pull request aims to correct them.
  The correction code has been tested against latest version (at the time of creation of this pull request) of: `langchain==0.0.336` `dashvector==1.0.6` .
- **Issue:** No issue is created for this.
- **Dependencies:** No dependency is required for this change,

- **Twitter handle:** `zeyanglin`

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/dashvector.py [hunk 1]
```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
     Example:
         .. code-block:: python
 
-            from langchain.vectorstores import dashvector
+            from langchain.vectorstores import DashVector
```
Comment (linzeyang) [inline libs/langchain/langchain/vectorstores/dashvector.py:32]: the imported `dashvector` had conflic with the other `dashvector` import below, and was inconsistent with the `DashVector(...)` code below.

File: libs/langchain/langchain/vectorstores/dashvector.py [hunk 2]
```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
...
             from langchain.embeddings.openai import OpenAIEmbeddings
             import dashvector
 
-            client = dashvector.Client.init(api_key="***")
-            client.create("langchain")
+            client = dashvector.Client(api_key="***")
```
Comment (linzeyang) [inline libs/langchain/langchain/vectorstores/dashvector.py:36]: `dashvector.Client.init(...)` is invalid code as of `dashvector==1.0.6`.
`dashvector.Client(...)` works.

File: libs/langchain/langchain/vectorstores/dashvector.py [hunk 3]
```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
...
+            client.create("langchain", dimension=1024)
```
Comment (linzeyang) [inline libs/langchain/langchain/vectorstores/dashvector.py:37]: `dimension` is a mandatory parameter to `create()` as of `dashvector==1.0.6`

File: libs/langchain/langchain/vectorstores/dashvector.py [hunk 4]
```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
...
             collection = client.get("langchain")
             embeddings = OpenAIEmbeddings()
-            vectorstore = Dashvector(collection, embeddings.embed_query, "text")
+            vectorstore = DashVector(collection, embeddings.embed_query, "text")
```
Comment (linzeyang) [inline libs/langchain/langchain/vectorstores/dashvector.py:40]: `Dashvector` seems like a typo. it should be `DashVector`

**Comment by baskaryan** [comment]
thank you @linzeyang!

**Pull request merged**